docs/FAQ/how-to-manually-configure-ssl-using-cert-manger-in-bold-bi-kubernetes-deployment.md

```diff
@@ -0,0 +1,109 @@
+# How TO Manually Configure SSL Using Cert Manager in Bold BI Kubernetes Deployment
```

Comment:
How to Manually Configure SSL Using Cert Manager in Bold BI Kubernetes Deployment

Reply:
concern addressed